src/routes/offline.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+</script>
+
+<section>
+	<h1>OFFLINE PAGE</h1>
```

Comment:
Blijkt nog niet helemaal optimaal te werken. Als ik in de **Application** tab _offline_ aan zet en de app gebruikt.

Cachen op spelpagina's werkt. Ga ik terug naar home dan krijg ik een _failed to fetch_. Ga ik naar spel dat niet gecached is, dan gewoon _no internet_. Klopt dat? Denk dat je het wel kan reproduceren.

Met name het _failed to fetch_ wil je niet.

Reply:
Klopt, deze branch is meer voor de setup. Als deze gemerged is gaan we de fetch werkend maken